raiden-dapp/src/services/raiden-service.ts

```diff
@@ -373,6 +389,46 @@ export default class RaidenService {
     } as NotificationPayload);
   }
 
+  private async notifyChannelSettleSuccess(
+    partner: string,
+    tokenNetwork: string,
+    txHash: string
+  ) {
+    let description = i18n.t('notifications.settlement.success.description', {
+      partner,
+      tokenNetwork,
+      txHash,
+    });
+
+    if (this._subkey) {
+      description +=
+        ' ' + i18n.t('notifications.settlement.success.withdrawal_hint');
+    }
+    await this.store.dispatch('notifications/notify', {
+      title: i18n.t('notifications.settlement.success.title'),
+      description,
+      context: NotificationContext.INFO,
+      importance: NotificationImportance.HIGH,
+    } as NotificationPayload);
```

Comment:
`as` is a type assertion, it may hide errors.. You may leave them here, since this is the pattern already accepted, but this should be improved in the future, either by having these action payloads typesafe (not easy right now with vuex, should be possible on Vue3) or by declaring the payload beforehand with `: NotificationPayload` type, then TS will complain if the object literal doesn't match the expected type.

Reply:
Actually typing get removed on Vuex > 4.? So maybe we should then take some time to make the whole store stuff properly typed. I have there some ideas. :grimacing: